dataproc/create_cluster/create_cluster.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2019 Google LLC
```

Comment:
Filepath comment: looks like this is the first Dataproc sample. Could we put all of the snippets into the top-level `dataproc` directory and put the quickstart in a `dataproc/quickstart` directory? No need to duplicate `dataproc` in the name.

Also, if there will be additional cluster samples in the future (create, delete, list, etc.), a pattern I've liked for other samples is starting the filename with the name of the thing -- `cluster_create.go`, `cluster_delete.go`, etc. Not mandatory at all, but can make it easier to find the right file for each sample.

Reply:
Moved create_cluster to top-level but leaving file names as-is to match method names / file names in other languages.